profiles/package.mask

```diff
@@ -30,6 +30,14 @@
 
 #--- END OF EXAMPLES ---
 
+# Craig Andrews <candrews@integralblue.com> (22 Jun 2016)
```

Comment:
As far as I know, normal users cannot alter this file even by means of a PR. You need to either get a developer to approve this change and use his name instead OR use the name of a group such as "Gentoo Java team <java@gentoo.org>" in this case and get us to sign off on it.